src/benchmark/file_io_write_micro_benchmark.cpp

```diff
@@ -223,7 +213,7 @@ void FileIOWriteMicroBenchmarkFixture::mmap_write_benchmark(benchmark::State& st
 }
 
 BENCHMARK_DEFINE_F(FileIOWriteMicroBenchmarkFixture, IN_MEMORY_WRITE)(benchmark::State& state) {  // open file
-  const uint32_t NUMBER_OF_BYTES = state.range(0) * MB;
+	const auto NUMBER_OF_BYTES = static_cast<uint32_t>(state.range(0) * MB);
```

Comment:
some indentations are off, I know its a draft PR. Just want to put a final format run on the todo list :)